src/InterfaceMoiCalculator.cc

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "sdf/InterafaceMoiCalculator.hh"
+#include "sdf/Mesh.hh"
+#include "sdf/Element.hh"
+
+using namespace sdf;
+
+class InterfaceMoiCalculator::Implementation
+{
+  /// \brief Density of the mesh
+  public: double density;
```

Comment:
```suggestion
  public: double density{0.0};
```

Reply:
Added in [a929478](https://github.com/gazebosim/sdformat/pull/1304/commits/a9294787abba97f45c2247a94755f1e0c853c3b0)